Authors: Daniel Baumann <daniel@debian.org>
	 Rogério Theodoro de Brito <rbrito@ime.usp.br>
Description: Fixes typo in manpages.
 The patch also deals with hyphens used as minus signs.

Index: dvdisaster/documentation/dvdisaster.cs.1
===================================================================
--- dvdisaster.orig/documentation/dvdisaster.cs.1	2012-03-06 11:06:06.000000000 +0900
+++ dvdisaster/documentation/dvdisaster.cs.1	2012-03-06 11:12:03.263041664 +0900
@@ -49,8 +49,8 @@
 Obecné příklady použití:
 .TP
 .B \-r, \-\-read
-Načte obraz média na pevný disk. Použijte -rn-m k načtení určitého rozsahu sektorů,
-t.j. -r100-200.
+Načte obraz média na pevný disk. Použijte \-rn-m k načtení určitého rozsahu sektorů,
+t.j. \-r100-200.
 .TP
 .B \-c, \-\-create
 Vytvoří informaci .ecc pro obraz média.
@@ -98,7 +98,7 @@
 automaticky přidá souborové přípony .img a .ecc.
 .TP
 .B \-\-cache-size n
-velikost varovnávací paměti v MB v průběhu akce -c (výchozí: 32 MB).
+velikost varovnávací paměti v MB v průběhu akce \-c (výchozí: 32 MB).
 .TP
 .B \-\-dao
 předpokládá disk DAO; neořezávat konec obrazu.
Index: dvdisaster/documentation/dvdisaster.en.1
===================================================================
--- dvdisaster.orig/documentation/dvdisaster.en.1	2012-03-06 11:08:20.000000000 +0900
+++ dvdisaster/documentation/dvdisaster.en.1	2012-03-06 11:12:03.263041664 +0900
@@ -104,8 +104,8 @@
 Action selection (at least one action must be specified):
 .TP
 .B \-r, \-\-read
-Read the medium image to hard disc. Use -rn-m to read a certain sector range,
-e.g. -r100-200.
+Read the medium image to hard disc. Use \-rn-m to read a certain sector range,
+e.g. \-r100-200.
 .TP
 .B \-c, \-\-create
 Create .ecc information for the medium image.
@@ -213,7 +213,7 @@
 automatically add .iso and .ecc file suffixes.
 .TP
 .B \-\-cache-size n
-image cache size in MB during -c mode (default: 32MB).
+image cache size in MB during \-c mode (default: 32MB).
 .TP
 .B \-\-dao
 assume DAO disc; do not trim image end.
@@ -235,12 +235,12 @@
 continue reading after potentially fatal error condition.
 .TP
 .B \-\-internal-rereads n
-internal read attempts for defective CD media sectors (default: -1)
+internal read attempts for defective CD media sectors (default: \-1)
 .RS
 The drive firmware usually retries unreadable sectors a few times before
 giving up and returning a read error. It is more efficient to set this to 0 or 1
-and manage read attempts through the --read-attempts parameter. Most drives ignore
-this setting anyways. Use -1 to leave the drive at its default setting.
+and manage read attempts through the \-\-read-attempts parameter. Most drives ignore
+this setting anyways. Use \-1 to leave the drive at its default setting.
 .RE
 .TP
 .B \-\-query-size n
@@ -253,7 +253,7 @@
 selects raw reading mode for CD media (default: 20)
 .RS
 The recommended mode is 20, which makes the drive apply its built-in error 
-correction to the best possible extent before transfering a defective sector.
+correction to the best possible extent before transferring a defective sector.
 However some drives can only read defective sectors using mode 21, skipping
 the last stage of the internal error correction and returning 
 the uncorrected sector instead.
Index: dvdisaster/documentation/dvdisaster.de.1
===================================================================
--- dvdisaster.orig/documentation/dvdisaster.de.1	2012-03-06 11:08:20.000000000 +0900
+++ dvdisaster/documentation/dvdisaster.de.1	2012-03-06 11:12:03.263041664 +0900
@@ -112,8 +112,8 @@
 Aktions-Auswahl (mindestens eine Aktion muss gew\[:a]hlt werden):
 .TP
 .B \-r, \-\-read
-Datentr\[:a]ger-Abbild auf Festplatte einlesen. -rn-m liest einen Bereich von
-Sektoren, z.B. -r100-200.
+Datentr\[:a]ger-Abbild auf Festplatte einlesen. \-rn-m liest einen Bereich von
+Sektoren, z.B. \-r100-200.
 .TP
 .B \-c, \-\-create
 Erzeugt Fehlerkorrektur-Daten f\[:u]r das Abbild.
@@ -243,12 +243,12 @@
 Lesen nach m\[:o]glicherweise schwerwiegenden Fehlern fortsetzen.
 .TP
 .B \-\-internal-rereads n
-Leseversuche innerhalb des Laufwerks f\[:u]r besch\[:a]digte CD-Sektoren (Standard: -1)
+Leseversuche innerhalb des Laufwerks f\[:u]r besch\[:a]digte CD-Sektoren (Standard: \-1)
 .RS
 Laufwerke unternehmen normalerweise mehrere Versuche um einen besch\[:a]digten Sektor 
 zu lesen. Es ist typischerweise g\[:u]nstiger diesen Wert auf 0 oder 1 zu setzen
-und die Anzahl der Leseversuche \[:u]ber den Parameter --read-attempts zu steuern.
-Viele Laufwerke ignorieren diese Einstellung ohnehin. Benutzen Sie den Wert -1 um 
+und die Anzahl der Leseversuche \[:u]ber den Parameter \-\-read-attempts zu steuern.
+Viele Laufwerke ignorieren diese Einstellung ohnehin. Benutzen Sie den Wert \-1 um
 die Standardeinstellungen des Laufwerks zu verwenden.
 .RE
 .TP
@@ -260,7 +260,7 @@
 .RE
 .TP
 .B \-\-raw-mode n
-"Raw"-Lese-Verfahren f\[:u]r besch\[:a]digte CD-Sektoren (default: 20)
+"Raw"\-Lese-Verfahren f\[:u]r besch\[:a]digte CD-Sektoren (default: 20)
 .RS
 Das empfohlene Verfahren ist 20, bei dem das Laufwerk die eingebaute Fehlerkorrektur
 so weit wie m\[:o]glich anwendet, bevor es einen besch\[:a]digten Sektor 
@@ -277,7 +277,7 @@
 liest den gesamten Datentr\[:a]ger bis zu n-mal.
 .TP
 .B \-\-read-raw
-liest in der "raw"-Betriebsart sofern m\[:o]glich.
+liest in der "raw"\-Betriebsart sofern m\[:o]glich.
 .TP
 .B \-\-speed-warning n
 warnt bei Geschwindigkeits\[:a]nderung um mehr als n Prozent.
Index: dvdisaster/documentation/dvdisaster.it.1
===================================================================
--- dvdisaster.orig/documentation/dvdisaster.it.1	2012-03-06 11:06:06.000000000 +0900
+++ dvdisaster/documentation/dvdisaster.it.1	2012-03-06 11:12:03.267041662 +0900
@@ -43,8 +43,8 @@
 Tipici esempi di utilizzo:
 .TP
 .B \-r, \-\-read
-Estrae l'immagine del supporto sull'hard disc. Usare -rn-m per leggere un
-determinato intervallo di settori, ad es. -r100-200.
+Estrae l'immagine del supporto sull'hard disc. Usare \-rn-m per leggere un
+determinato intervallo di settori, ad es. \-r100-200.
 .TP
 .B \-c, \-\-create
 Crea l'informazione .ecc per l'immagine del supporto.
@@ -92,7 +92,7 @@
 aggiungi automaticamente i suffissi .img e .ecc.
 .TP
 .B \-\-cache-size n
-Dimensione della cache in MB con la modalit\[`a] -c (Standard: 32MB).
+Dimensione della cache in MB con la modalit\[`a] \-c (Standard: 32MB).
 .TP
 .B \-\-dao
 Utilizza la modalit\[`a] DAO; Il disco non sar\[`a] finalizzato.
